app/views/user_mailer/challenge_assignment_notification.html.erb

```diff
@@ -30,17 +28,17 @@
     <%= index+1 %>. <%= style_bold(prompt.title) %>
 
     <p>
-      <% if fandoms then %><%= style_bold("Fandom:") %> <%= fandoms %><br><% end %>      
-      <% if chars then %><%= style_bold("Characters:") %> <%= chars %><br><% end %>
-      <% if ships then %><%= style_bold("Relationships:") %> <%= ships %><br><% end %>
-      <% if ratings then %><%= style_bold("Rating:") %> <%= ratings %><br><% end %>
-      <% if warnings then %><%= style_bold("Warnings:") %> <%= warnings %><br><% end %>
-      <% if categories then %><%= style_bold("Category:") %> <%= categories %><br><% end %>
-      <% if atags then %><%= style_bold("Additional Tags:") %> <%= atags %><br><% end %>
-      <% if otags then %><%= style_bold("Optional Tags:") %> <%= otags %><br><% end %>
-      <% if prompt.url && !prompt.url.blank? then %><%= style_bold("Prompt URL:") %> <%= style_link(prompt.url, prompt.url) %><br><% end %>
+      <% if fandoms then %><%= style_bold("#{t('activerecord.models.fandom', count: tag_groups["Fandom"].count)}:") %> <%= fandoms %><br><% end %>      
```

Comment:
Should we have a set of models _withcolon in the activerecord section, then, or add them specifically for this email? It seems to be the kind of thing we'd use in multiple places.